sjsonnet/src-jvm-native/sjsonnet/SjsonnetMain.scala

```diff
@@ -7,8 +7,24 @@ import java.nio.file.NoSuchFileException
 import scala.util.Try
 import scala.util.control.NonFatal
 
+// JsonnetWorker (in universe) extends this trait in the universe so that it can pass the cache based on Caffeine to main0 here
+trait ParseCacheInterface {
```

Comment:
Let's call this `ParseCache` and leave `Interface` out of the name, it's simpler and there's no confusion around what it is.